CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v12.0.0 30th November 2023
+
+### Changed
+- Added multi-database support [#522](https://github.com/gocardless/statesman/pull/522)
```

Comment:
I think this needs to include something along the lines of: "use the ActiveRecord connection of the transaction class etc", as that is the actual change. The multi-db support is more of a side effect of using the appropriate connection.

Do we need to update any documentation? Eg. any caveats users need to be aware of?

Reply:
Good shout, updated. The usage and interface of statesman should be exactly the same from a model/transition point of view - the only thing that has changed is the connection used when creating and updating transitions.

Reply:
Nope, only internal changes to behaviour detection methods.